README.md

```diff
@@ -106,6 +106,18 @@ The HomeGraph API is used to report the state of your devices to Google and to r
 1. Start using the Google Assistant.
 
 
+#### Integrating Google Sign-In
+
+The Google Sign-In feature allows to login using the Google credential. See [Integrating Google Sign-In into your web app](https://developers.google.com/identity/sign-in/web/sign-in) for more datails.
+
+
+1. Navigate to the [Google Cloud Console API & Services page](https://console.cloud.google.com/apis/credentials)
+2. Click Create credentials > OAuth client ID.
+3. Select the Web application application type.
+4. Name your OAuth 2.0 client and click Create
+
+After configuration is complete, take note of the client ID that was created. You will need the client ID to complete the configuration.
```

Comment:
I had to add my hostname (https://example.com:3001) as authorized JavaScript origin on the Google Credentials page. Maybe you can add this to the Readme.

Reply:
Done, thanks.